src/model/skeleton.py

```
@@ -248,10 +248,10 @@ def forward_memory_only(self, input_ids: Tensor) -> Tuple[Tensor, Tensor]:
             if not hasattr(block, "memory"):
                 continue
             mem = block.memory
+            # Paper-compliant 2-projection GELU MLP: M(x) = x + W1(σ(W2(x)))
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `σ` (GREEK SMALL LETTER SIGMA). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF003)